apps/picture_renditions.py

```diff
@@ -33,9 +33,12 @@ def create(self, docs, **kwargs):
             doc['renditions'] = renditions
             ids.append(item['_id'])
 
+            is_article_edit_media = doc.get('is_article_edit_media', False)
+
             updates = {'renditions': item['renditions']}
             try:
-                get_resource_service('archive').update(item['_id'], updates, item)
+                if not is_article_edit_media:
+                    get_resource_service('archive').update(item['_id'], updates, item)
```

Comment:
This is going to change the `_etag`. Is the client aware about the change in `_etag`?

Reply:
This is to avoid the change in `_etag`. If the media is in the body of the article then only this update method gets called otherwise not.

Reply:
@GyanP instead having another parameter in the payload and update the resource over here can you issue another request from the client to update resource `item` ?

Reply:
@mdhaman moved the resource update request to the client.